libraries/AP_Mount/AP_Mount_Siyi.cpp

```diff
@@ -112,6 +112,7 @@ void AP_Mount_Siyi::update()
             break;
         case MountTargetType::RATE:
             send_target_rates(mnt_target.rate_rads.pitch, mnt_target.rate_rads.yaw, mnt_target.rate_rads.yaw_is_ef);
+            update_angle_target_from_rate(mnt_target.rate_rads, mnt_target.angle_rad);
```

Comment:
I think the angle targets will not be completely accurate because the gimbal does not perfectly obey our rate requests.  I think this PR is still an improvement though so I'm not going to hold it up over this.

An alternative would be to use the angles reported by the gimbal but I'm not sure it's worth it.